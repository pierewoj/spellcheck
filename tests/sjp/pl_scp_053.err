1,25 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
3,17 Euclid: 
5,21 Przechowawcze: przechowawce
5,61 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
5,251 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
5,450 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
5,553 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
9,7 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
9,107 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
9,163 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
11,57 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
15,1 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
38,617 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
