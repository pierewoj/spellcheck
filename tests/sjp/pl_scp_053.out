Identyfikator podmiotu: #SCP-PL-053

Klasa podmiotu: #Euclid

Specjalne Czynności #Przechowawcze: Z racji nietypowej formy #SCP-PL-053, obiekt nie jest praktycznie możliwy do zabezpieczenia w racjonalnym okresie. Fundacja aktualnie lokalizuje, a następnie zasypuje, zamurowuje lub w inny sposób blokuje wejścia do #SCP-PL-053, a pozostawia wyłącznie jedno na dany kraj. W Polsce wykorzystywane jest wejście zabezpieczone jako Ośrodek Magazynowy 97; pozostałe są w trakcie wyłączania. Eksploracje i eksperymenty na #SCP-PL-053 mogą prowadzić naukowcy z upoważnieniem co najmniej poziomu 2., za zgodą menedżera projektu #SCP-PL-053. To samo dotyczy oddziałów zagranicznych.

Fundacja na bieżąco jest informowana w zakresie zarządzania prawami miejskimi wszystkich krajów świata.

Opis: #SCP-PL-053 to sieć podziemnych kanałów, hipotetycznie łącząca wszystkie istniejące na Ziemi miasta. #SCP-PL-053 został odkryty podczas poszukiwania wejść do #SCP-PL-043 w polskich Tatrach.

Fundacja stosuje specyficzne nazewnictwo wobec zjawiska #SCP-PL-053: główny kanał, przebiegający po niezidentyfikowanym równoleżniku, niemający początku ani końca oraz kanaliki (kanały odbiegające od głównego kanału; prowadzące do wyjść na powierzchnię), z których każdy jest ponumerowany zgodnie z widniejącą obok wejścia doń legendą.

Promień światła kanału wynosi od 10 do 12 metrów. Wysokość od lustra wody do części sufitowej kanału jest różna, aczkolwiek głębokość wody zawsze wynosi, w przybliżeniu, 4 metry. W kanalikach głębokość wody jest taka sama, jak w kanale, aczkolwiek promień światła (jak dotąd wiadomo) waha się pomiędzy 6 a 7 metrami. Uznano, że woda jest słodka i zdatna do picia, a pływające w niej ryby (karasie) można w razie potrzeby konsumować. W kanale i kanalikach panuje ciemność, a temperatura, bez względu na to, przez jaki region świata dany kanalik przebiega, wynosi od 24 do 26 stopni Celsjusza.

#SCP-PL-053 dotyczą trzy znane anomalie:

    Niemożliwe jest określenie głębokości ani lokalizacji kanałów, jeśli pomiar dokonywany jest spoza wejścia do kanalika. Próby przewiercenia potencjalnych miejsc, przez które kanał lub kanaliki przepływają, są bezskuteczne — pracownicy dowiercają się do litej skały tam, gdzie według obserwatorów wewnętrznych znajdować miał się kanał lub kanalik;
    W momencie uzyskania przez nowo powstałe miasto praw miejskich, obserwuje się rozlegające się w kanale dźwięki, przypominające wiercenie świdrem oraz stukanie młotem, które trwają, w zależności od fizycznej odległości miasta od równoleżnika, od dwóch do piętnastu dni — źródła tych dźwięków nie rozpoznano, aczkolwiek, po ustaniu hałasu, w lokalizacji odpowiadającej rejonowi, w którym owo, nowo powstałe miasto istnieje, znaleźć można nowy kanalik. (Podobną, analogiczną anomalię, obserwuje się po odebraniu praw miejskich miastom oraz po fuzjach miast).
    Kanaliki zdaje się dotykać anomalia przestrzenna: dla przykładu fragment kanału, który przebiega przez rejon polski, zawiera dwa istniejące w odległości 1 metra od siebie kanaliki, które, będąc jednakowej długości (około 11 kilometrów), prowadzą do Sopotu i Zielonej Góry, co fizycznie jest niemożliwe.

Wyjście z kanalika na powierzchnię może znajdować się w opustoszałym lub nieopustoszałym domu, toalecie na dworcu autobusowym czy w kanałach miejskich. Niekiedy cywile odkrywają owe wyjścia, nie mogąc jednak bez odpowiedniego sprzętu dotrzeć do głównego kanału; część takich osób umiera, stanąwszy w wodzie (szczegóły w dodatku A).

Wejście do kanału oznaczone jest po jednej ze ścian numerem kanalika, a po drugiej — legendą (namalowane czerwoną farbą), która przedstawia zapisane w lokalnym języku miasta ich numery oraz to, w którą stronę podróżujący powinien się kierować po wyjściu z kanalika, by dotrzeć do celu (lewo czy prawo), i jaka odległość dzieli go od celu. Mimo że sam kanał jest położony wokół kuli ziemskiej, według obecnych szacunków, jest długości zaledwie ok. 8600 kilometrów.

Dodatek A: "Regulamin" podróżowania, zapisany przy każdej legendzie, w języku miasta, do którego dany kanalik prowadzi.

    Regulamin

    Drodzy podróżnicy, prosimy o zważanie na trzy zasady:

    1. Woda, którą będą się państwo poruszać, jest zdatna do picia, a pływające nią stworzenia stanowić mogą zdrowy posiłek bez gotowania. Prosimy jednak nie oddawać do tej wody moczu ani do niej nie defekować.
    2. Prosimy nie próbować pływać ani wchodzić do wody. To niebezpieczne.
    3. Prosimy nie przeszkadzać innym podróżnikom.

    Prosimy pamiętać, by zabrać ze sobą łódź, lampę, wędkę, kubeł, nóż i widelec oraz worki na śmieci.
    Dziękujemy za uwagę. Życzymy miłej podróży.

Fundacja zdecydowała zbadać rezultat złamania drugiej zasady. Po upuszczeniu pracownika klasy "D" do wody, jego ciało momentalnie zesztywniało, a lekarz, po wciągnięciu zwłok na łódź, stwierdził zgon testera. Obecnie, nie wiadomo, w jaki sposób doszło do śmierci testera, aczkolwiek zakłada się, iż pod wodą mogą znajdować się urządzenia filtrujące, a w jednym z takowych mogło dojść do śmiertelnego dla testera przebicia, kiedy ten przebywał w wodzie. Hipotezy tej jednak nie zweryfikowano, a kolejna próba jej zbadania również zakończyła się natychmiastową śmiercią testera (mimo że przeprowadzona w innym rejonie #SCP-PL-053). Zauważono jednak, że dopóki część ciała podróżnika znajduje się na lądzie (zazwyczaj skale) bądź w łodzi, dopóty zachowane jest jego bezpieczeństwo.

Fundacja aktualnie jest zdania, iż punkt nr 3 odnosi się do potencjalnych podróżujących, jako że — pomijając ciekawskich cywilów — nie odnaleziono w sieci kanałów ani jednego podróżnika. Pomimo zaś łamania wielokrotnie, w celach eksperymentalnych, zasady pierwszej, nie odnotowano z tego powodu żadnych negatywnych konsekwencji, pomijając samo zanieczyszczanie wody.
